panic: runtime error: index out of range

syncthing-inotify v0.8.3 from apt.syncthing.net:
panic: runtime error: index out of range
goroutine 42 [running]:
panic(0x7cd0a0, 0xc82000e060)
/usr/local/go/src/runtime/panic.go:481 +0x3e6
main.watchSTEvents(0xc8202960f0, 0xc82027e200, 0x9, 0x9)
/home/buildslave/workspace/syncthing-inotify-debian/src/github.com/syncthing/syncthing-inotify/syncwatcher.go:906 +0x5ea
created by main.main
/home/buildslave/workspace/syncthing-inotify-debian/src/github.com/syncthing/syncthing-inotify/syncwatcher.go:288 +0x5a4


syncthing-inotify 0.8.4:
panic: runtime error: index out of range

goroutine 39 [running]:
panic(0x670280, 0xc42000c160)
	/home/pi/dev/tools/go/src/runtime/panic.go:500 +0x1a1
main.watchSTEvents(0xc420257ef0, 0xc420258000, 0x9, 0x9)
	/home/pi/dev/src/github.com/syncthing/syncthing-inotify/syncwatcher.go:920 +0x15c
created by main.main
	/home/pi/dev/src/github.com/syncthing/syncthing-inotify/syncwatcher.go:283 +0x396

--- a/syncwatcher.go
+++ b/syncwatcher.go
@@ -933,7 +933,7 @@ func watchSTEvents(stChans map[string]chan STEvent, folders []FolderConfiguratio
 			time.Sleep(configSyncTimeout)
 			continue
 		}
-		if events == nil {
+		if len(events) == 0 {
 			continue
 		}
 		for _, event := range events {

